mlrun/db/httpdb.py

```diff
@@ -647,6 +647,87 @@ def get_pipeline(self, run_id: str, namespace: str = None, timeout: int = 10):
 
         return resp.json()
 
+    def create_feature_set(
+        self, feature_set: Union[dict, schemas.FeatureSet], project="", versioned=False
+    ) -> schemas.FeatureSetCreateOutput:
+        project = project or default_project
+        path = f"projects/{project}/feature_sets"
+        params = {"versioned": versioned}
+
+        if isinstance(feature_set, dict):
+            feature_set = schemas.FeatureSet(**feature_set)
+
+        name = feature_set.metadata.name
+        error_message = f"Failed creating feature-set {project}/{name}"
+        resp = self.api_call(
+            "POST",
+            path,
+            error_message,
+            params=params,
+            body=json.dumps(feature_set.dict()),
+        )
+        return schemas.FeatureSetCreateOutput(**resp.json())
+
+    def get_feature_set(
+        self, name: str, project: str = "", tag: str = None, uid: str = None
+    ) -> schemas.FeatureSet:
+        project = project or default_project
+        reference = uid or tag or "latest"
```

Comment:
add condition that raises `MLRunInvalidArgumentError` if both uid and tag are given

Reply:
same on update

Reply:
👍